packages/calypso-storybook/package.json

```
@@ -22,7 +22,8 @@
 	"devDependencies": {
 		"@automattic/calypso-eslint-overrides": "workspace:^",
 		"@automattic/calypso-typescript-config": "workspace:^",
-		"@storybook/addon-viewport": "^7.6.20"
+		"@storybook/addon-themes": "7.6.20",
+		"@storybook/addon-viewport": "^7.6.19"
```

Let's align with the rest of the storybook dependencies used across the repo

```suggestion
		"@storybook/addon-viewport": "^7.6.20"
```